### PR TITLE
ATI for STY pages

### LINKED ISSUE #5421: ATI analytics for STYs
**Is your feature request related to a problem? Please describe.**
We want to add the ability to track ATI analytics for STYs on page load.

**Describe the solution you'd like**
- Add STY logic to ATIAnalytics container.
- Verify `containers/ATIAnalytics/params/cpsAssetPage/buildParams.js` is constructing the right object for STY assets.
- Add relevant test coverage.

Note: This issue should be completed before https://github.com/bbc/simorgh/issues/5407.

When I go to a story (STY) page, then a pageload event fires to ATI that includes the values below:

Based on example STY: https://www.bbc.com/mundo/noticias-internacional-51266689
AReS feed: https://ares-broker.api.bbci.co.uk/api/asset/mundo/noticias-internacional-51266689

ATI Label | ATI Code | Value | Source
-- | -- | -- | --
Countername/ Page | - | mundo.world.story.51266689.page
("counterName" in payload)  | AReS
Destination | 598342 | WS_NEWS_LANGUAGES (on live) | Simorgh
  | 598343 | WS_NEWS_LANGUAGES_TEST (on test) | Simorgh
Producer  Level 2 | 62 | Mundo | Simorgh
Custom Site Variables |   |   |  
Content ID | - | urn:bbc:content:assetUri:mundo/noticias-internacional-51266689 (urn:bbc:cps:curie - this is in AReS payload in "curie", which is the GUID for the content: "http://www.bbc.co.uk/asset/faf2993a-dcc0-0746-81a6-a6aca4cd59b1" | AReS
App Type | - | responsive OR amp | Simorgh
App name | - | news-mundo | Simorgh
Language | - | es | AReS
URL | - | https://www.bbc.com/mundo/noticias-internacional-51266689 | Simorgh
Content Type | - | article | This needs to be set
Page Title | - | Defined as {title} - BBC News Mundo where content title is Promo Headline from AReS payload | AReS for title / Simorgh for Brand
Referrer URL (x6) | - | The referrer URL, where available. Not set otherwise | Should be set by Library
Publication Date | - | [1580366149000](firstPublished but needs to be converted from unix time) | AReS
Updated Date | - | [1580512096857](lastUpdated but needs to be converted from unix time) | AReS
LDP Things (x13) | - | aresResponse.metadata.tags.about[i].thingLabel  separated by ~ | AReS
LDP Things IDs (x14) | - | aresResponse.metadata.tags.about[i].thingId    separated by ~E.g. 0239ab33-1cfc-4f5d-babb-a8159711af3e~0f469e6a-d4a6-46f2-b727-2bd039cb6b53 | AReS
Campaign | - | [WS - Give me perspective] (Can be multiple values separated by ~  see: Campaigns / campaignName in payload)  | AReS
Category | - | [Explainer] (defined as categoryName in the payload)  | AReS
Library Version | - | simorgh | TBC

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
- https://github.com/bbc/simorgh/issues/1310
- https://github.com/bbc/simorgh/issues/5230

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #5421 

**Overall change:** 
Enables ATI tracking for STY pages and updates PGL test to not use STY fixture data.

**Code changes:**
- Add STY logic to get ATI url in  ATIAnalytics container.
- Removed STY fixture asset in PGL test
- updated relevant snapshot due to changes

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added labels to this PR for the relevant pod(s) affected by these changes
- [X] I have assigned this PR to the Simorgh project

**Testing:**
- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [X] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/CpsPgl/index.test.jsx [hunk 1]
```diff
@@ -145,22 +144,22 @@ describe('CPS PGL Page', () => {
 
   it('should only render firstPublished timestamp for Igbo when lastPublished is less than 1 min later', async () => {
     const pageData = await preprocessor(
-      igboPageData,
+      pglAboutData,
```
Comment (rhenshaw56) [inline src/app/pages/CpsPgl/index.test.jsx:147]: `igboPageData` is an  STY asset not PGL. This change ensures the ATI logic for STY's isn't used in a PGL test.

**Review by andrewscfc — APPROVED** [review]
Checked this out and ran against: http://localhost:7080/mundo/noticias-internacional-51266689

image

All fields match values in issue: https://github.com/bbc/simorgh/issues/5421#issue-560551503

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Comment by paruchurisilpa** [comment]
Checked this using : http://localhost:7080/mundo/noticias-internacional-51266689
LGTM..

**Pull request merged**